price-estimator/src/filter.rs

```diff
@@ -161,13 +146,15 @@ fn estimated_best_ask_price_filter(
         .and(warp::query::<QueryParameters>())
 }
 
-async fn get_markets<T>(
+async fn get_markets(
     market: Market,
     _query: QueryParameters,
-    orderbook: Arc<Orderbook<T>>,
+    orderbook: Arc<Orderbook>,
 ) -> Result<impl Reply, Infallible> {
+    // TODO: should this route make use of the rounding buffer? This wasn't the case with the
+    // previous rounding buffer.
```

Comment:
TODO

Reply:
The issue I see with providing an orderbook with rounding buffers applied is that you place an order for 1ETH for 200DAI it will show up as an order for 1ETH for 199.9999DAI which is a little counter intuitive.